### PR TITLE
Jetpack Cloud: Improve ActivityCard and realtime backup view

### PR DESCRIPTION
**Pull request by rcoll**
#### Changes proposed in this Pull Request

* Updates styles and some form on the ActivityCard component
* Applies styles to the Realtime context for backups
* One easy bug fix to backup detail page

#### Testing instructions

* Load up the backup section for a site with realtime backups enabled.
* Check that styles are applied to the activity cards for a day with activities. They should roughly reflect the Figma design.
* Check that you are able to perform a restore and download a backup for an activity.
* Check that the detail page works.

### CODE REVIEW

File: client/landing/jetpack-cloud/components/activity-card/index.jsx [hunk 1]
```diff
@@ -12,15 +14,36 @@ import { Card } from '@automattic/components';
...
+	}
+
+	togglePopoverMenu = () => this.setState( { showPopoverMenu: ! this.state.showPopoverMenu } );
+	closePopoverMenu = () => this.setState( { showPopoverMenu: false } );
+	createRestoreUrl = restoreId => `/backups/${ this.props.siteSlug }/restore/${ restoreId }`;
+	createDownloadUrl = downloadId => `/backups/${ this.props.siteSlug }/download/${ downloadId }`;
+	createDetailUrl = backupId => `/backups/${ this.props.siteSlug }/detail/${ backupId }`;
+	triggerRestore = () => page.redirect( this.createRestoreUrl( this.props.activity.rewindId ) );
+	triggerDownload = () => page.redirect( this.createDownloadUrl( this.props.activity.rewindId ) );
+	triggerDetails = () => page.redirect( this.createDetailUrl( this.props.activity.rewindId ) );
```
Comment (cleacos) [inline client/landing/jetpack-cloud/components/activity-card/index.jsx:41]: We could use the helper functions that Caleb implemented in `jetpack-cloud/sections/backups/paths.ts`

Comment (rcoll) [reply client/landing/jetpack-cloud/components/activity-card/index.jsx]: - Paths fixed in https://github.com/Automattic/wp-calypso/pull/40867/commits/d1e1014faf87cd52c7a94ecb6c263c778737ccb6
- Styling fixed in https://github.com/Automattic/wp-calypso/pull/40867/commits/963c05e7d6e37618038ebaa2ef6bb18acf7c78a2

Thanks for the feedback :-)

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3213110

Hi @rcoll, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Your site is backed up in real time (as you make changes) as well as in one daily backup.`
- `There were no changes on this day. Your daily backup is above.`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3213110).

**Pull request merged**